test.js

```diff
@@ -1 +1,11 @@
-console.log('Hello World')
+console.log('Hello World');
+
+console.log('Hello Universe');
+
+console.log('Hello other stuff');
+
+if(foo === "bar") {
+  console.log('stuff')
+}
+
+console.log('will it trigger the others?');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.dev.codacy.org/gh/troubleshoot-codacy/no-suggestions-2/pullRequest?prid=30889)
```suggestion
console.log("will it trigger the others?");
```